packages/types/src/cart/common.ts

```diff
@@ -1,4 +1,479 @@
+   */
+  tax_lines?: ShippingMethodTaxLineDTO[]
+  /**
+   * The associated adjustments.
+   *
+   * @expandable
+   */
+  adjustments?: ShippingMethodAdjustmentLineDTO[]
+
+  /**
+   * When the shipping method was created.
+   */
+  created_at: Date
+  /**
+   * When the shipping method was updated.
+   */
+  updated_at: Date
+
+  original_total: number
+  original_subtotal: number
+  original_tax_total: number
+
+  total: number
+  subtotal: number
+  tax_total: number
+  discount_total: number
+  discount_tax_total: number
+}
+
+export interface CartLineItemDTO {
+  /**
+   * The ID of the line item.
+   */
+  id: string
+  /**
+   * The title of the line item.
+   */
+  title: string
+  /**
+   * The subtitle of the line item.
+   */
+  subtitle?: string
+  /**
+   * The url of the line item thumbnail.
+   */
+  thumbnail?: string
+  /**
+   * The line item quantity
+   */
+  quantity: number
+  /**
+   * The product ID of the line item.
+   */
+  product_id?: string
+  /**
+   * The product title of the line item.
+   */
+  product_title?: string
+  /**
+   * The product description of the line item.
+   */
+  product_description?: string
+  /**
+   * The product subtitle of the line item.
+   */
+  product_subtitle?: string
+  /**
+   * The product type of the line item.
+   */
+  product_type?: string
+  /**
+   * The product collection of the line item.
+   */
+  product_collection?: string
+  /**
+   * The product handle of the line item.
+   */
+  product_handle?: string
+  /**
+   * The variant ID of the line item.
+   */
+  variant_id?: string
+  /**
+   * The variant sku of the line item.
+   */
+  variant_sku?: string
+  /**
+   * The variant barcode of the line item.
+   */
+  variant_barcode?: string
+  /**
+   * The variant title of the line item.
+   */
+  variant_title?: string
+  /**
+   * The variant option values of the line item.
+   */
+  variant_option_values?: Record<string, unknown>
+  /**
+   * Whether the line item requires shipping or not
+   */
+  requires_shipping: boolean
+  /**
+   * Whether the line item is discountable or not
+   */
+  is_discountable: boolean
+  /**
+   * Whether the line item price is tax inclusive or not
+   */
+  is_tax_inclusive: boolean
+  /**
+   * The original price of the item before an adjustment or a sale.
+   */
+  compare_at_unit_price?: number
+  /**
+   * The price of the item
+   */
+  unit_price: number
+  /**
+   * The associated tax lines.
+   *
+   * @expandable
+   */
+  tax_lines?: LineItemTaxLineDTO[]
+  /**
+   * The associated adjustments.
+   *
+   * @expandable
+   */
+  adjustments?: LineItemAdjustmentLineDTO[]
+  /**
+   * Holds custom data in key-value pairs.
+   */
+  metadata?: Record<string, unknown> | null
+  /**
+   * When the line item was created.
+   */
+  created_at?: Date
+  /**
+   * When the line item was updated.
+   */
+  updated_at?: Date
+
+  compare_at_total?: number
+  compare_at_subtotal?: number
+  compare_at_tax_total?: number
```

Comment:
question: what are those? I ve asked on notion some times ago but I didn't get the answer ahah